src/components/layout/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import logo from "../../../utils/logo.svg";
+import "./NavBarStyles.css";
+
+class NavBar extends React.Component {
+    render() {
+        return (
+          <div><h1>NavigationBar</h1> <span><img src={logo} /></span></div>
```

Comment:
Formatting is important for good quality code. If our code is of good quality than it will help us and other collaborators to work on the code easily. :)